docs/component-wrapper-helper.md

```diff
@@ -65,6 +66,9 @@ The component wrapper includes several methods to make managing options easier,
   aria_attributes ||= {}
   role ||= nil
 
+  # margin_bottom will be applied by default, use this line to set a different default
```

Comment:
Maybe it's worth rewording this a bit, until I read the ruby code I thought this meant that every component using the wrapper will get a margin bottom by default

Reply:
Thanks, have removed. That was a hangover from an earlier attempt at this.